cmd/buildkitd/main_containerd_worker.go

```diff
@@ -208,6 +213,9 @@ func applyContainerdFlags(c *cli.Context, cfg *config.Config) error {
 	if c.GlobalIsSet("containerd-cni-config-path") {
 		cfg.Workers.Containerd.NetworkConfig.CNIConfigPath = c.GlobalString("containerd-cni-config-path")
 	}
+	if c.GlobalIsSet("containerd-netns-pool-size") {
```

Comment:
With the current implementation isn't it more like a `cni-pool-size`?

Reply:
Renamed.